tests/HTMLPurifierTest.hack

```diff
@@ -103,12 +103,19 @@ class HTMLPurifierTest extends HackTest {
 </table>';
 		$purifier = new HTMLPurifier\HTMLPurifier($config);
 		$clean_html = $purifier->purify($dirty_html);
-		expect($clean_html)->toEqual('<table><caption>
+		expect($clean_html)->toEqual('<table>
+  <caption>
     Cool table
   </caption>
-  <tfoot><tr><th>I can do so much!</th>
```

Comment:
I think changing this breaks the test, right? :laughcry:

Reply:
Hmm, very strange that test failed. When I run it using my local hhvm, they passed, but in container version of hhvm it failed.

Reply:
Yeah, this is the issue we've been talking about in the Hacklang workspace.  Something strange happens with the spacing between versions or between Mac/Linux